lib/utils.ts

```diff
@@ -1,6 +1,36 @@
-import { type ClassValue, clsx } from "clsx"
-import { twMerge } from "tailwind-merge"
+import { type ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
 
 export function cn(...inputs: ClassValue[]) {
-  return twMerge(clsx(inputs))
+  return twMerge(clsx(inputs));
+}
+
+export function formatDate(date: Date): string {
+  const month = (new Date(date).getMonth() + 1).toString().padStart(2, "0"); // Add 1 because months are 0-indexed, pad with leading zero
+  const day = new Date(date).getDate().toString().padStart(2, "0"); // Pad with leading zero
+  const year = new Date(date).getFullYear();
+
+  return `${month}/${day}/${year}`;
+}
+
+export function formatCurrency(number: number): string {
+  return new Intl.NumberFormat("en-US", {
+    style: "currency",
+    currency: "USD",
+    minimumFractionDigits: 0,
+    maximumFractionDigits: 0,
+  }).format(number / 100);
```

Comment:
Just wondering why are you inputing the payment per day in cents? wouldn't that be a bit confusing for the facility to always remember to put the payment when making a job post in cents